src/Traversals/depthfirst.jl

```diff
@@ -93,23 +93,26 @@ end
 end
 
 
-@traitfn function topological_sort(g::AG::IsDirected, alg::DepthFirst=DepthFirst()) where {T, AG<:AbstractGraph{T}}
+"""
+    topological_sort(g, alg=DepthFirst())
+
+Return a [topological sort](https://en.wikipedia.org/wiki/Topological_sorting) of a directed graph `g`
+using [`TraversalAlgorithm`](@ref) `alg` as a vector of vertices in topological order.
+"""
```

Comment:
don"t we need the doc block on a function signature as in:
function topological_sort end

Reply:
great catch. yes.